expo/ntid
# Convert to Typescript and export TS definitions

# Why

This is a dependency of many TypeScript projects, so it would be nice to have TypeScript definitions. Since it is very small and well-tested, the easiest way to generate these is to just convert the library to use typescript instead of babel.

# How

Convert stuff to TypeScript, add typescript config and jest typescript stuff.

# Test Plan

1. `yarn tsc`
1. `yarn jest`

Manual tests:
1. `yarn link` it to a project that uses it
1. `yarn ntid example` results in `example[QrOec5Ba4AK7fpqp2SPVfy]` (to check that CLI works)
1. `yarn tsc` to check that exported type definitions work.
